sky/utils/command_runner.pyi

```diff
@@ -0,0 +1,99 @@
+import enum
```

Comment:
Same as above - if `command_runner.py` includes all typing info, is this still necessary? Would be nice if we can minimize redundancy in typing information and keep it at one place

Reply:
Same as above. We need to overload the method type hint to determine the return type based on the argument `require_outputs`

Reply:
Same as above. We need to overload the method type hint to determine the return type based on the argument `require_outputs`